sagemaker-debugger/xgboost_census_explanations/xgboost-census-debugger-rules.ipynb

```diff
@@ -7,7 +7,7 @@
    },
```

Comment:
<p>same suggestions as above. The plot is more readable if the name of each curve is the actual feature name. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>fixed</p>